### PR TITLE
Finish lint for client/components/duplicates

### PR DESCRIPTION
**Pull request by ZeHiro**

### CODE REVIEW

File: client/components/duplicates/index.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                similar.push([op, next]);
+            j += 1;
+        }
+    }
+
+    debug(`${similar.length} pairs of similar operations found`);
+    debug(`findRedundantPairs took ${Date.now() - before}ms.`);
+    // The duplicates are sorted from last imported to first imported
+    similar.sort((a, b) => Math.max(b[0].dateImport, b[1].dateImport) -
+        Math.max(a[0].dateImport, a[1].dateImport)
```
Comment (bnjbvr) [inline client/components/duplicates/index.js]: This is a bit ugly to read. Could each call to Math.max be on its own line, with the minus at the end of the first line?

File: client/components/duplicates/index.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            pairs: []
+        };
+        this.listener = this._listener.bind(this);
+    }
+
+    _listener() {
```
Comment (bnjbvr) [inline client/components/duplicates/index.js]: Please remove _listener into listener.

File: client/components/duplicates/index.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+        let sim;
+        if (pairs.length === 0) {
+            sim = <div> { $t('client.similarity.nothing_found') } </div>;
+        } else {
+            sim = pairs.map(p => {
+                let key = p[0].id.toString() + p[1].id.toString();
+                return <SimilarityPairComponent key={ key } a={ p[0] } b={ p[1] }/>;
+            });
+        }
```
Comment (bnjbvr) [inline client/components/duplicates/index.js]: Please add a new line after the else

File: client/components/duplicates/index.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        store.removeListener(State.accounts, this.listener);
+        store.removeListener(State.operations, this.listener);
+    }
+
+    render() {
+        let pairs = this.state.pairs;
+
+        let sim;
+        if (pairs.length === 0) {
+            sim = <div> { $t('client.similarity.nothing_found') } </div>;
```
Comment (bnjbvr) [inline client/components/duplicates/index.js]: Can we remove the spaces just after the opening tag and before the closing tag?

File: client/components/duplicates/index.js [hunk 5]
```diff
@@ -0,0 +1,107 @@
+import { store, State } from '../../store';
+import { debug as dbg, translate as $t } from '../../helpers';
+import SimilarityPairComponent from './item';
```
Comment (bnjbvr) [inline client/components/duplicates/index.js]: Can simply be renamed "Pair", here and in the 'item' file.

**Comment by bnjbvr** [comment]
Nice, thanks!

File: client/components/duplicates/index.js (hunk 1 above)
Comment (ZeHiro) [inline client/components/duplicates/index.js]: done

File: client/components/duplicates/index.js (hunk 2 above)
Comment (ZeHiro) [inline client/components/duplicates/index.js]: done

File: client/components/duplicates/index.js (hunk 3 above)
Comment (ZeHiro) [inline client/components/duplicates/index.js]: done

File: client/components/duplicates/index.js (hunk 4 above)
Comment (ZeHiro) [inline client/components/duplicates/index.js]: done

File: client/components/duplicates/index.js (hunk 5 above)
Comment (ZeHiro) [inline client/components/duplicates/index.js]: done

**Comment by bnjbvr** [comment]
Rebased and pushed as https://github.com/bnjbvr/kresus/commit/fe198e0b7cfda31cd2f550eb88517da1f9303859 , thanks!

**Pull request closed without merging**